### ISSUE TITLE
package upgrades break styleguidist

### ISSUE THREAD

**Issue by stinoga** [styleguidist/react-styleguidist#254]
Styleguidist has been working great for me first off. Thanks for the awesome tool!

I recently updated some npm packages, and my styleguide now breaks with this error:
```
style-guide.bundle.js:24 Uncaught Error: Cannot find module "styleguide!index.js"
```

Here are the package changes I made:
```json
-    "babel-plugin-module-alias": "^1.5.0",
+    "babel-plugin-module-alias": "^1.6.0",
-    "istanbul-instrumenter-loader": "^0.1.3",
+    "istanbul-instrumenter-loader": "^1.0.0",
-    "babel-core": "^6.8.0",
-    "babel-eslint": "^6.0.4",
-    "babel-loader": "^6.2.4",
+    "babel-core": "^6.18.2",
+    "babel-eslint": "^7.1.1",
+    "babel-loader": "^6.2.8",
-    "babel-plugin-transform-runtime": "^6.8.0",
-    "babel-polyfill": "^6.8.0",
-    "babel-preset-es2015": "^6.6.0",
-    "babel-preset-react": "^6.5.0",
-    "babel-preset-stage-2": "^6.5.0",
-    "babel-runtime": "^6.6.1",
+    "babel-plugin-transform-runtime": "^6.15.0",
+    "babel-polyfill": "^6.16.0",
+    "babel-preset-es2015": "^6.18.0",
+    "babel-preset-react": "^6.16.0",
+    "babel-preset-stage-2": "^6.18.0",
+    "babel-runtime": "^6.18.0",
-    "eslint": "^3.2.0",
-    "eslint-loader": "^1.5.0",
+    "eslint": "^3.10.2",
+    "eslint-loader": "^1.6.1",
-    "invariant": "^2.2.1",
+    "invariant": "^2.2.2",
-    "webpack": "^1.12.2"
+    "webpack": "^1.13.3"
```

Thanks!

**Comment by sapegin** [comment]
I don’t see anything suspicious. Have you tried to remove `node_modules`, do `npm cache clean` and reinstall everything again?

**Comment by stinoga** [comment]
@sapegin Yep. I did all those steps, to no avail. Another note is that I don't see the issue until I run `npm shrinkwrap` effectively locking in these versions.

**Comment by stinoga** [comment]
@sapegin So I've found the culprit, although not sure what the issue is. When running `styleguide-server` I get the below error:
```
ERROR in Unexpected token {
 @ ./~/react-styleguidist/src/index.js 40:18-48 81:1-84:3
```

Any ideas?

**Comment by sapegin** [comment]
Missed loader or something like that. Maybe json-loader. Unless I have a demo project, it’s very hard to guess.

I hope upcoming 5.0.0 release will be much harder to break like that ;-)

**Comment by stinoga** [comment]
@sapegin What's the timing on that? Is there a branch? Maybe I could pull it down and test.

**Comment by sapegin** [comment]
Relatively soon I hope. It's in the next branch. It's better to use ne webpackConfig option instead of old updateWebpackConfig, it's safer.

**Comment by stinoga** [comment]
Got it! Upgrading node to version `6.3.0` fixed the problem.

**Comment by O4epegb** [comment]
Had same issue with node 5.10. This one #250 too. I'm not able update node at work just yet, so decided to try other version of styleguidist, 4.0.0, build static just fine, but server still crashing with same error.

**Comment by sapegin** [comment]
@O4epegb It would be awesome if you can help up investigate it. Because we support Node 4+ and we run tests on Travis on 4, 6 and 7. And we probably missing something. I can’t reproduce it.

**Comment by sapegin** [comment]
Should be fixed in 4.4.3.

**Comment by O4epegb** [comment]
Very nice, thank you for good work!
